index.html

```diff
@@ -38,8 +40,130 @@ <h2>I am a Software Developer</h2>
+          </ul>
+          <button>See Project</button>
+        </section>
+        <section class="card card4">
+          <h1>Professional Art<br>Printing data</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+        <section class="card card5">
+          <h1>Professional Art<br>Printing data</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+        <section class="card card6">
+          <h1>Professional Art<br>Printing data</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+      </section>
+    </section>
+
+    <section class="about_me">
+      <section class="description">
+        <h1>About me</h1>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      </section>
+      <section class="resume">
+        <button>Get My Resume</button>
+        <a class="about_bgimg"></a>
+      </section>
+      <hr>
+      <section class="competence">
+        <ul>
+          <li><div class="lang"><img src="images_2/diamond.png">Language</div>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </li>
+        </ul>
+        <ul>
+          <li><div class="lang"><img src="images_2/square.png">Frameworks</div>
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </li>
+        </ul>
+        <ul>
+          <li><div class="lang"><img src="images_2/circle.png">Skills</div>
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </li>
+        </ul>
```

Comment:
- [ ] In This section, the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`.